zulip_bots/zulip_bots/provision.py

```diff
@@ -35,6 +36,7 @@ def provision_bot(path_to_bot, force):
 
 
 def parse_args(available_bots):
+    # type: (Iterator[AnyStr]) -> argparse.Namespace
```

Comment:
What's the reason for using AnyStr instead of str here?

Reply:
I was just trying to get rid of the errors below. You're right, `str` works fine here. Fixed.